docs/team/repository-code-standards.md

```diff
@@ -0,0 +1,36 @@
+# Repository Management and Code Standards
+
+JupyterHub projects should take advantage of pre-commit and tools to help maintain consistent formatting within a repo to improve overall code quality, review efficiency, and readability of code.
+Sensible use of pre-commit to run formatting tools and code linters can add consistency and improve maintainability.
+
+## Preferred tools and pre-commit hooks
+
+The team has found the following tools and their pre-commit hooks to be useful. The following pre-commit hooks can be added to any repo when convenient, though please communicate with anyone who has an open pull request if it will lead to major conflicts:
+
+  - [black](https://black.readthedocs.io/)
```

Comment:
worth recommending to just use default settings for these whenever possible?

Reply:
Added to the `Configuration of a tool` section